students/TinaB/lessonTwo_TB/lesson2.py

```diff
@@ -0,0 +1,104 @@
+#manually create box
+def printing():
+     plus = '+ '
+     minus = '- '
+     pipe = '| '
+     space = ' '
+     print(plus + (minus*4) + plus + (minus*4) + plus)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(plus + (minus*4) + plus + (minus*4) + plus)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(pipe + space*8 +pipe+ space*8 +pipe)
+     print(plus + (minus*4) + plus + (minus*4) + plus)
+
+
+def dynamicprinting(x):
```

Comment:
Python convention is to use 'snake case' for function and variable names with words separated by underscores e.g. `dynamic_printing`